admin/app/components/complementary-certifications/attach-badges/badges/index.js

```diff
@@ -47,6 +52,7 @@ export default class Badges extends Component {
         const targetProfileBadges = targetProfile.badges?.map((badge) => ({
           id: badge.id,
           label: badge.title,
+          isCertifiable: badge.isCertifiable,
```

Comment:
Ce fichier étant spécifique au rattachement de profil cibles dans le cadre des complémentaires, je propose de filtrer directement à l'init (c'est àa ou le fetch est réalisé)

```suggestion
        const targetProfileBadges = targetProfile.badges?
        .filter(this.badges, (badge) => badge.isCertifiable)
        .map((badge) => ({
          id: badge.id,
          label: badge.title,
          isCertifiable: badge.isCertifiable,
```

Reply:
A discuter, mais ayant une tendance API first, qu'est ce que l'équipe pense d'implémenter un query param sur la route actuellement appelée par le `findRecord` ?

Ember semble [gérer cela nativement](https://guides.emberjs.com/v2.11.0/models/relationships/#toc_retrieving-related-records) du style: `store.findRecord('post', {include: 'all'});`

A voir si on rentre dans le débat habituel : c'est admin et le coût de cette modif est + grande.

Reply:
Suite discussions tripartite à l'oral, je résume mon ordre de préférence (préférence purement technique) :

* ajouter le filter [selon recommendations JSONAPI](https://www.jsonapi.net/usage/reading/filtering.html) sur l'API existante

Reply:
done le query param, histoire de pas le décaler à plus tard